### ISSUE TITLE
where are files?

### ISSUE THREAD

**Issue by brw31415** [creationix/git-browser#13]
hi, i am running the chromebook app version. where is the default dir for downloading files?

**Comment by creationix** [comment]
I don't believe this ever downloaded files to the filesystem.  It just synced the git database locally using indexed db.  The actual files are stored in git blob entries in that db.
